test/basic.test.ts

```diff
@@ -205,6 +205,42 @@ describe('pages', () => {
     await expectNoClientErrors('/?test=true')
   })
 
+  it('/optional/[[opt]].vue', async () => {
```

Comment:
instead, would you update `packages/nuxt/test/pages.test.ts`? I think we can cover this in a unit test.

Reply:
Added unit tests. Do you insist to replace `fixtures/basic` ones? They also checks the ordering of routes as well.

Reply:
Thank you ❤️

I think the route ordering should be tested by vue-router rather than us. The e2e tests here are good but I think we can save the CI time.